build.gradle

```diff
@@ -35,7 +35,7 @@ ext {
     targetSdkVersion = 28
     compileSdkVersion = 28
     buildToolsVersion = '26.0.2'
-    kotlinVersion = "1.3.21"
+    kotlinVersion = "1.3.61"
```

Comment:
Updated as the previous one was quite outdated and I was not fully sure about all the features of kt version 1.3.71 .